packages/promotion/src/utils/compute-actions/order.ts

```diff
@@ -0,0 +1,19 @@
+import { PromotionTypes } from "@medusajs/types"
+import {
+  ApplicationMethodAllocation,
+  ApplicationMethodTargetType,
+} from "@medusajs/utils"
+import { getComputedActionsForItems } from "./items"
+
+export function getComputedActionsForOrder(
+  promotion: PromotionTypes.PromotionDTO,
+  itemApplicationContext: PromotionTypes.ComputeActionContext,
+  itemIdPromoValueMap: Map<string, number>
+): PromotionTypes.ComputeActions[] {
+  return getComputedActionsForItems(
+    promotion,
+    itemApplicationContext[ApplicationMethodTargetType.ITEMS],
+    itemIdPromoValueMap,
+    ApplicationMethodAllocation.ACROSS
+  )
```

Comment:
general question: Do we have risk between a cart and an order to apply a different amount if the application method gets updated in the mean time?

Reply:
Possible, but that can be handled by the consumer as an added validation before processing the order by comparing the snapshots.